runtime/runtime/src/actions.rs

```diff
@@ -189,6 +189,8 @@ pub(crate) fn action_function_call(
         .into());
     }
     state_update.trie.request_code_recording(account_id.clone());
+    #[cfg(feature = "test_features")]
+    apply_recorded_storage_garbage(function_call, state_update);
     let mut receipt_manager = ReceiptManager::default();
     let mut runtime_ext = RuntimeExt::new(
         state_update,
```

Comment:
One thing I'm missing is a way to say "all witness should now be 20MB in size"
We can send a lot of receipts, but that'll include multiple receipts into a chunk, which will go above the size.
Maybe it'd be possible to accept only one special receipt per chunk? Maybe save it until a special node hash?

Reply:
good idea with using a special node hash, implemented that in 4146a0fe8